index.html

```diff
@@ -333,23 +329,25 @@ <h3 class="skill-title">
         
     </div>
     </section>
-    <footer class="contact-me">
+    <footer class="contact-me" id="contact-me">
         <div class="contact-me-div">
-            <h2 class=contact-head>Contact me</h2>
+            <h3 class=contact-head>Contact me</h3>
             <p class="contact-description">
                 If you have an application you are interested 
                 in developing, a feature that you need built 
                 or a project that needs coding. I’d love to 
-                help with it.
+                help with it 
             </p>
-            <form id="form" action="https://formspree.io/f/mjvjrozl" method="POST">
+            <form id="form" action="https://formspree.io/f/1" method="POST">
                 <input type="text" name="name" id="name" class="name-email" placeholder="Enter your name" maxlength="30" required>
                 <input type="email" name="email" id="email" class="name-email" placeholder="Enter your email" required>
                 <textarea name="message" id="message" rows="6" maxlength="500" placeholder="write your message here" required></textarea>
-                <button type="submit" class="button form-button" >Get it Touch</button>
+                <button type="submit" class="button" >Get it Touch</button>
             </form>
         </div>
     </footer>
```

Comment:
- [ ] Kindly note that you are required to follow the Figma design template.You must stick to the design as much as possible (e.g., font, colours, images, tests, margins). 
- The contact form has different colour compared to that in the design.
- The `Let's connect` button in the headline and the about section uses a different font.
- The `Languages, Frameworks and Skills` section also has some font issues and the spacings are a lot larger than the Figma design.
Kindly refer back to the design template and make all the necessary changes required.

Reply:
thanks for your review. The thing is I changed the contact form color because of the accessibility test. No problem I will put back the color from the template

Reply:
I changed the fonds of the let's connect too

Reply:
Oh I understand, just giving a report on the contrast issue would have been enough for the accessibility project. Good job tho. Thanks for making the changes.

Reply:
I also added some fonds and color to Languages, Frameworks and Skills as in the template